src/ImGui/Desktop/Fusee.ImGui.Desktop/Templates/ImGuiFilePicker.cs

```diff
@@ -264,15 +255,15 @@ public virtual unsafe void Draw(ref bool filePickerOpen)
             });
             if (Directory.Exists(currentFolder))
             {
-                CurrentOpenFolder = currentFolder;
+                CurrentOpenFolder = new DirectoryInfo(currentFolder);
             }
             else if (File.Exists(currentFolder))
             {
                 var fi = new FileInfo(currentFolder);
```

Comment:
This is unclear to me: File.Exits with a folder should always return false?
And `currentFolder` is `CurrentOpenFolder` which should never contain a file name.

Reply:
Yes. This is wrong.